.gitignore

```diff
@@ -25,3 +25,5 @@ Dockerfile.cross
 *.swp
 *.swo
 *~
+
+/local-dev
```

Comment:
Add `config/featureToggles/.gitignore` with line for `config/featureToggles/featureToggles.local.yaml`
And add a new line at the end of the file